inlong-sort/sort-common/src/main/java/org/apache/inlong/sort/protocol/node/Node.java

```diff
@@ -75,6 +76,7 @@
         @JsonSubTypes.Type(value = TubeMQExtractNode.class, name = "tubeMQExtract"),
         @JsonSubTypes.Type(value = RedisExtractNode.class, name = "redisExtract"),
         @JsonSubTypes.Type(value = DorisExtractNode.class, name = "dorisExtract"),
+        @JsonSubTypes.Type(value = InfluxDBExtractNode.class, name = "influxExtract"),
```

Comment:
Please keep the naming consistent of `influxExtract`  between the Node and ExtractNode.